include/zmq.h

```diff
@@ -265,6 +265,9 @@ ZMQ_EXPORT int zmq_msg_init_data (zmq_msg_t *msg, void *data,
     size_t size, zmq_free_fn *ffn, void *hint);
 ZMQ_EXPORT int zmq_msg_send (zmq_msg_t *msg, void *s, int flags);
 ZMQ_EXPORT int zmq_msg_recv (zmq_msg_t *msg, void *s, int flags);
+ZMQ_EXPORT int zmq_recv_peer (void *s, void *zid, int zid_len, void *buf,
```

Comment:
Please move new APIs in the DRAFT section of the header (and duplicate them in src/zmq_drafts.h)

Reply:
I started putting it into `zmq_drafts.h`, but then for sockopts I had to define them in in `zmq.h` (otherwise I had to add includes to `zmq_drafts.h` all over). Please advise how to proceed better for getsockopts.

Reply:
All definitions are duplicated - once in the include/zmq.h inside the \#ifdef DRAFT below, and once in src/zmq_drafts.h  where they are all inside a \#if not defined (DRAFT). These way you can have the definitions available internally even if the library is not built by the user with DRAFT APIs.